### PR TITLE
Adding linux label to test file

### PR DESCRIPTION
**Pull request by rajasec**
Added the linux compile flag for unit test
Fixed some minor typos.

Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

File: libcontainer/cgroups/utils_test.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// +build linux
```
Comment (crosbymichael) [inline libcontainer/cgroups/utils_test.go]: I think you need a new line after the build tag to make it consistent

**Comment by rajasec** [comment]
@crosbymichael 
Updated as per your comments.

**Comment by LK4D4** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**